portfolio/src/main/java/com/google/sps/data/Log.java

```diff
@@ -0,0 +1,17 @@
+package com.google.sps.data;
+
+/** A Login Info Collection */
+public final class Log {
```

Comment:
Created a log class like my comment class

Reply:
nit:  "Log" unfortunately could have different meanings in programming.  I would suggest renaming to make it clearer what this class holds.  Maybe something like "LoggedInStatus" or "AuthStatus" could work well.